src/dbg/_exports.cpp

```diff
@@ -169,7 +169,7 @@ static bool getLabel(duint addr, char* label)
             duint start;
             if(FunctionGet(addr, &start, nullptr) && addr == start)
             {
-                sprintf_s(label, MAX_LABEL_SIZE, "sub_%" fext "X", start);

```

Comment:
Yes, it needs more fine tuning. Also the race condition is a major concern
that could lead to crash. By the way, could you check the previous PR so we
can get rid of those irrelavant diffs?